tools/POTools/Services/Resources/ResXFileReader.cs

```diff
@@ -19,10 +19,10 @@ public ResXFileReader(TextReader reader)
             _root = (root?)xmlSerializer.Deserialize(xmlReader) ?? new root { Items = Array.Empty<object>() };
     }
 
-    public IEnumerator<KeyValuePair<string, string>> GetEnumerator()
+    public IEnumerator<(string name, string value, string comment)> GetEnumerator()
```

Comment:
Same issue as above.

Reply:
Same as above, kept for extra clarity. If you 'insist', I'll remove the extra comment, but I do prefer named tuple fields over KeyValuePair, even if it's only two fields...

Reply:
Just to be clear, I meant tuple member casing.

I don't mind tuples, but nowadays readonly record structs may be an even better option.